Gemfile

```diff
@@ -6,6 +6,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 # Specify your gem's dependencies in tramway.gemspec.
 gemspec
 
+gem 'anyway_config'
+
```

Comment:
why you need empty lines in gemfile?

Reply:
No need, you are right! Removed :slightly_smiling_face: